_data/apps.yml

```diff
@@ -1848,3 +1848,12 @@
     - "programming"
     - "productivity"
   icon: "cycligent.png"
+-
+  name: "Mixmax"
+  description: "See every action on your emails in real-time. Compose anywhere."
+  website: "https://mixmax.com/download"
+  keywords:
+    - "email"
+    - "analytics"
+  icon: "mixmax.png"
+  featuredOnHomepage: true
```

Comment:
This line needs to be removed and is causing the test failure (we aren't updating the homepage apps at this time).